packages/rich-text/src/plugins/EmbeddedEntityBlock/LinkedEntityBlock.tsx

```diff
@@ -51,8 +52,6 @@ export function LinkedEntityBlock(props: LinkedEntityBlockProps) {
     Transforms.removeNodes(editor, { at: pathToElement });
   };
 
-  // TODO: fixme -- props not available on new editor
-  const isDisabled = false; // editor.props.readOnly || editor.props.actionsDisabled;
```

Comment:
Out of curiosity, What did we use `actionsDisabled` for in the past?

Reply:
We use it for the `ReadOnlyRichTextEditor` in the UI diff view: https://github.com/contentful/user_interface/blob/09590e62f7ccf4058b10efad175573ceb1077532/src/javascripts/app/widgets/RichText/ReadOnlyRichTextEditor.js#L14-L25

But that's in coordination with `readOnly` and I can't remember if there was a practical rather than semantic reason for separating between these two.